setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+version = attr: pystac.version.__version__
```

Comment:
Would you be OK to add a trailing newline to files without them? Git and all the \*nix tools expect it.

Reply:
Fixed in c48c15bf8537eb81dbf28003eb225806bf55d3ff